erpnext/setup/setup_wizard_slide/welcome_to_erpnext!/welcome_to_erpnext!.json

```diff
@@ -7,13 +7,13 @@
  "domains": [],
  "help_links": [],
  "idx": 0,
- "image_src": "/assets/erpnext/images/illustrations/onboard.png",
+ "image_src": "/assets/erpnext/images/illustrations/desk-onboard.png",
```

Comment:
Also, I feel, the first slide's image can be more welcoming rather than just showing the desk illustration?

Reply:
The desk illustration is more relatable than the handshake before. Besides the Desk UI is synonymous to ERPNext within the group that knows about Frappe and ERPNext. I don't see anything fitting other than this. Would love alternate recommendations for the illustrations.

The copy is welcoming enough on the other hand.